modules/combat.py

```diff
@@ -80,9 +84,9 @@ def combat_logic_wrapper(self):
                 Utils.touch_randomly(self.region["menu_button_battle"])
                 Utils.wait_update_screen(1)
                 continue
-            if Utils.find("combat/menu_fleet") and (lambda x:x > 414 and x < 584)(Utils.find("combat/menu_fleet").y):
+            if Utils.find("combat/menu_fleet") and (lambda x: 414 < x < 584)(Utils.find("combat/menu_fleet").y):
```

Comment:
I don't like this change, it looks really obnoxious to me.

Reply:
alright, will revert.